openjdk/build.xml

```diff
@@ -75,7 +92,15 @@
 						<property name="jdkName" value="openj9-openjdk-jdk${JDK_VERSION}" />
 					</else>
 				</if>
-				<property name="regressionRepo" value="https://github.com/ibmruntimes/${jdkName}.git" />
+                                <if>
+                                    <contains string="${PLATFORM}" substring="zos"/>
+                                    <then>
+                                        <property name="regressionRepo" value="git@github.com:ibmruntimes/${jdkName}.git" />
```

Comment:
You don't need if platform zos here, you can just use ssh (https://github.com/smlambert/openjdk-tests/blob/zos/openjdk/build.xml#L83) for all platforms, as I believe it does work for all of them.

Reply:
I wasn't sure if anyone using git@ to build/test would need a GPG key setup? which not every user may do...?

Reply:
In order to use ssh you need an ssh key setup or in the case of Jenkins nodes an ssh-agent wrapped around whatever calls this build.xml. Further, I thought on z/os we needed to use the git:// protocol?